torch/nn/_functions/dropout.py

```diff
@@ -58,8 +58,9 @@ def backward(ctx, grad_output):
 class FeatureDropout(Dropout):
 
     @staticmethod
-    def symbolic(input, p=0.5, train=False, inplace=False):
-        return None
+    def symbolic(g, input, p=0.5, train=False, inplace=False):
+        # NB: No FeatureDropout in ONNX, use Dropout instead.
+        return Dropout.symbolic(g, input, p, train, inplace)
```

Comment:
Dropout and FeatureDropout are different. You’re silently exporting a wrong function

Reply:
https://github.com/onnx/onnx/issues/113 We are discussing whether we should add FeatureDropout to ONNX. We should either add FeatureDropout or output some warning information and export FeatureDropout as a non-op.

Reply:
If there’s a Bernoulli sampling op then you can implement feature dropout with sample + broadcast

Reply:
No sampling op... We will see the results of the discussion.

Reply:
I think it's ok to merge it if it's used in Eval mode, but it should probably be an error or at least a warning in train mode.

Reply:
It's a good idea to raise an error if train=True.